posix.mak

```diff
@@ -266,12 +266,15 @@ OBJS = $(addsuffix $(DOTOBJ),$(addprefix $(ROOT)/,$(C_MODULES)))
 
 MAKEFILE = $(firstword $(MAKEFILE_LIST))
 
+# build with shared library support
```

Comment:
Incomplete comment.

     # Build with shared library support on Linux and FreeBSD.